packages/playground/package.json

```diff
@@ -12,22 +12,29 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@former-ui/preset-nuxt-ui": "workspace:*",
+    "@nuxt/ui": "^3.3.4",
+    "@unhead/vue": "^1",
     "@vueuse/core": "12.0.0",
     "former-ui": "workspace:*",
-    "vue": "^3.5.13"
+    "vue": "^3.5.13",
+    "vue-router": "^4"
```

Comment:
@AmmarKaraja Please pin all depencies in the entire package json. This makes it safer...